app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
+  class DashboardController < ApplicationController
+    before_action :is_admin?
```

Comment:
Let's create an Admin::BaseController and  do `is_admin?` in that and inherit every admin controller from that controller